cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {defaultAction} from 'services/helpers';
+import {createStore, compose} from 'redux';
+
+const INTERVAL_OPTIONS = [
+  {
+    id: '5min',
+    value: 'Every 5 min'
+  },
+  {
+    id: '10min',
+    value: 'Every 10 min'
+  },
+  {
+    id: '30min',
+    value: 'Every 30 min'
+  },
+  {
+    id: 'Hourly',
+    value: 'Hourly'
+  },
+  {
+    id: 'Daily',
+    value: 'Daily'
+  },
+  {
+    id: 'Weekly',
+    value: 'Weekly'
+  },
+  {
+    id: 'Monthly',
+    value: 'Monthly'
+  },
+  {
+    id: 'Yearly',
+    value: 'Yearly'
+  }
+];
+
+const DAY_OF_WEEK_OPTIONS = _.range(0, 7);
```

Comment:
We should avoid directly using `_`. This will force webpack to use add the entire lodash library. 

It should be `import range from 'lodash/range'` and then just `range(0,7)`